docs/glossary.md

```diff
@@ -0,0 +1,7 @@
+---
```

Comment:
is this doc autogenerated?

Reply:
No. But I documented this on the README.md